packages/zone/src/heap.js

```diff
@@ -38,13 +35,15 @@ const detachedHeapStores = Far('heapStores', {
  */
 export const makeHeapZone = (baseLabel = 'heapZone') => {
   const { makeOnce, wrapProvider } = makeOnceKit(baseLabel, detachedHeapStores);
-  return Far('heapZone', {
+
+  const subZoneProvider = (label, _options) =>
```

Comment:
Please rename to be consistent with whatever naming convention you have for these encapsulated provider functions. I am not suggesting any deviation from those conventions.